pkgs/data/fonts/iosevka/comfy.nix

```diff
@@ -1,14 +1,18 @@
-{stdenv, lib, nodejs, nodePackages, remarshal, ttfautohint-nox, fetchurl}:
+{ callPackage, lib, fetchFromGitHub }:
 
 let
   sets = [ "comfy" "comfy-fixed" "comfy-duo" "comfy-wide" "comfy-wide-fixed" ];
-  privateBuildPlan = builtins.readFile ./comfy-private-build-plans.toml;
+  version = "0.2.1";
+  src = fetchFromGitHub {
+    owner = "protesilaos";
+    repo = "iosevka-comfy";
+    rev = version;
+    sha256 = "sha256-Ki3/8REmVb1XtvchdLZJ9Zc+EoP3LP9tvruuP/K1Xpo=";
+  };
+  privateBuildPlan = /. + ((builtins.unsafeDiscardStringContext src.outPath)
```

Comment:
Why `unsafeDiscardStringContext`?

Doesn't `${src}/private-build-plans.toml` work?

Not sure if this counts as IFD, does it matter how it's used in makeIosevkaFont down below?

If it has to fetch it to evaluate the package hydra won't be happy.

Reply:
> Doesn't `${src}/private-build-plans.toml work?`

If by that, you mean passed in qoutes, so it becomes a string, then no, unfortunately not.
The reason for this can be seen in the first commit of this PR. I am attempting to differentiate between a path and a string in order to not break anything when this variable has been used to directly pass in the buildplan as a string. Therefore I need to have it somehow as an actual path type. But ofborg ist not happy with it.

> Not sure if this counts as IFD, does it matter how it's used in makeIosevkaFont down below?

I dont think this is an IFD because everything can be prefetched. But it still complains that it cannot be evaluated "purely".

Reply:
What does IFD mean?

Reply:
https://nixos.wiki/wiki/Import_From_Derivation